resources/SmokeResource.py

```diff
@@ -1,4 +1,5 @@
 from flask_restful import Resource
+from ArtifyAPI.utils import socket_connect
 
```

Comment:
Need to rename modules, smoke_resource.py instead of SmokeResource.py, the same changes should be applied for  UploadPhotoResource.py